src/rpc/client/ts/src/change.ts

```diff
@@ -137,21 +137,21 @@ export function overwrite(
  * @param remove_bytes_count number of bytes to remove
  * @param replace replacement bytes
  * @return positive change serial number of the insert on success
- * @remarks if the bytes being replaced have the same length as the replacement
- * bytes, use overwrite for better efficiency
  */
 export function replace(
   session_id: string,
   offset: number,
   remove_bytes_count: number,
   replace: string | Uint8Array
 ): Promise<number> {
-  return new Promise<number>(async (resolve) => {
-    await pauseViewportEvents(session_id)
-    await del(session_id, offset, remove_bytes_count)
-    await resumeViewportEvents(session_id)
-    return resolve(await insert(session_id, offset, replace))
-  })
+  return replace.length == remove_bytes_count
```

Comment:
is there a difference w/ `==` vs `===` for turnary operations?

Reply:
Good find.  We should probably be using strict equals everywhere in TS/JS.  I'll make a ticket for that.  The test cases pass, but I did fix this instance though so we get this change merged.

Reply:
#494 created to cover using strict equals instead of equals for all TS/JS.